### PR TITLE
chore!: remove leading underscores cohere and anthropic

### PR DESCRIPTION
**Pull request by Askir**
BREAKING CHANGE: leading underscores have been removed from all cohere and anthropic function parameter names

### CODE REVIEW

File: projects/extension/sql/idempotent/004-cohere.sql [hunk 1]
```diff
@@ -41,13 +40,13 @@ set search_path to pg_catalog, pg_temp
 -------------------------------------------------------------------------------
 -- cohere_tokenize
 -- https://docs.cohere.com/reference/tokenize
-create or replace function ai.cohere_tokenize(_model text, _text text, _api_key text default null) returns int[]
+create or replace function ai.cohere_tokenize(model text, text text, api_key text default null) returns int[]
```
Comment (jgpruitt) [inline projects/extension/sql/idempotent/004-cohere.sql:43]: change `text` to `text_input`

File: projects/extension/sql/idempotent/004-cohere.sql [hunk 2]
```diff
@@ -74,23 +73,23 @@ set search_path to pg_catalog, pg_temp
 -- cohere_embed
 -- https://docs.cohere.com/reference/embed-1
 create or replace function ai.cohere_embed
-( _model text
-, _input text
-, _api_key text default null
-, _input_type text default null
-, _truncate text default null
+( model text
+, input text
```
Comment (jgpruitt) [inline projects/extension/sql/idempotent/004-cohere.sql:77]: change `input` to `input_text`

File: projects/extension/sql/idempotent/004-cohere.sql [hunk 3]
```diff
@@ -130,11 +129,11 @@ set search_path to pg_catalog, pg_temp
...
-( _model text
-, _inputs text[]
-, _api_key text default null
-, _examples jsonb default null
-, _truncate text default null
+( model text
+, inputs text[]
+, api_key text default null
+, examples jsonb default null
+, truncate text default null
```
Comment (jgpruitt) [inline projects/extension/sql/idempotent/004-cohere.sql:136]: should we change `truncate` to `truncate_long_inputs`?

File: projects/extension/sql/idempotent/003-anthropic.sql [hunk 4]
```diff
@@ -1,53 +1,52 @@
...
-, _top_k int default null
-, _top_p float8 default null
+( model text
+, messages jsonb
+, max_tokens int default 1024
+, api_key text default null
+, base_url text default null
+, timeout float8 default null
+, max_retries int default null
+, system text default null
```
Comment (jgpruitt) [inline projects/extension/sql/idempotent/003-anthropic.sql:12]: `system` to `system_prompt`?

**Review by jgpruitt — CHANGES_REQUESTED** [review]

**Review by jgpruitt — APPROVED** [review]

**Review by thedodd — APPROVED** [review]

**Pull request merged**